en/lessons/basics/documentation.md

```diff
@@ -68,6 +69,8 @@ iex> h Greeter
 Provides a function hello/1 to greet a human
 ```
 
+But if you're working in existing project, you don't need to include file manually, as we did with `c("greeter.ex")` in this example.
```

Comment:
I'm not entirely sure what we want to say here. Can you help me understand this a little better?

Reply:
I intended something like this:

> Line `c("greeter.ex")` is due to our current environment, you won't need it in a proper mix project.

Reply:
Any ideas how to rephrase it to sound good?

Reply:
Maybe we should include this as a "note" like we do in other lessons, something like:

```
_Note_: we don't need to manually compile our files as we did above if we're working within the context of a mix project.
```

How is this @elixirschool/english?

Reply:
Maybe we should add a note on running `iex -S mix` to load the IEx console for the current project if you're working in a mix project? We could explain that approach vs. the `load IEx, compile the file` approach described here for one-off Elixir files.